styles/globals.css

```diff
@@ -2,28 +2,34 @@
 @tailwind components;
 @tailwind utilities;
 
-
 @font-face {
-  font-family: "Neufile";
-  src: url('../public/assets/font/NeufileGrotesk-Regular.eot');
+  font-family: 'Neufile';
+  src: url('/assets/font/NeufileGrotesk-Regular.woff2');
+}
+
+html {
```

Comment:
Greats Job! However, to make it much better refer to this link which shows a very proper way of adding fonts in your 
in NextJs. here is the link: [https://nextjs.org/docs/basic-features/font-optimization ](url)